### ISSUE TITLE
Replication problem

### ISSUE THREAD

**Issue by LTC232** [sony/MambaPEFT#3]
Hello, thank you for open-sourcing the code. We noticed that during the reproduction process, there was an error with the LoRA method for the "Pythia" model. It seems the corresponding code was not provided.

**Comment by myoshimura1234** [comment]
Thank you for your attention.
When releasing the code, we removed the LoRA implementation for Transformer not to cause confusion between LoRA for Mamba and LoRA for Transformer.
 
To evaluate LoRA with Pythia, please copy all the code in
```
https://github.com/AGI-Edgerunners/LLM-Adapters/blob/main/peft/src/peft/tuners/lora.py
```
and paste it into
```
https://github.com/sony/MambaPEFT/blob/main/language/commonsense_reasoning/mamba_peft/src/peft/tuners/lora.py
```
Then you should be able to evaluate with the following command:
```
CUDA_VISIBLE_DEVICES=0 python finetune.py --base_model EleutherAI/pythia-160m --data_path commonsense_170k.json --adapter_name lora --output_dir ./results/pythia-160m_lora --learning_rate 1e-5

TASKS='boolq,social_iqa,hellaswag,piqa,arc_easy,arc_challenge,winogrande,openbookqa'
python lm_harness_eval.py --model hf --model_args "pretrained=./results/pythia-160m_lora,trust_remote_code=True" --output_path results/pythia-160m --tasks $TASKS
```
The best learning rate is listed in Table 8 in the paper.
